source/doc-pages/learn/adapters.md

```diff
@@ -1,5 +1,5 @@
 # Adapters
 
 * [how-to](/learn/adapters/how-to) - learn how to build your own adapter
-* [rom-sql](/learn/adapters/sql)
-* [rom-cassandra](/learn/adapters/cassandra)
+* [rom-sql](/learn/adapters/adapter/sql/)
+* [rom-cassandra](/learn/adapters/adapter/cassandra/)
```

Comment:
this is not right, could you please move sql & cassandra to adapters dir? my bad...